cli.js

```diff
@@ -22,6 +22,10 @@ if (!args.dir || (!args.all && (!args.platform || !args.arch))) {
   process.exit(1)
 }
 
+if (args.tmpdir === 'false') {
+  args.tmpdir = false
+}
+
```

Comment:
Should add a comment that minimist doesn't do this for us. I almost forgot why this conditional was added.